guides/common/modules/con_introduction-to-organization-and-location-context.adoc

```diff
@@ -0,0 +1,12 @@
+[id="Introduction_to_Organization_and_Location_Context_{context}"]
+= Introduction to Organization and Location Context in {Project}
+
+Context in {Project} consists of xref:Managing_Organizations_{context}[Organizations] and xref:Managing_Locations_{context}[Locations].
+
+Most entities in {Project} are associated with at least one organization and location context.
+Since entities and users can generally only access resources and other entities within their own context, organizations and locations form an integral part of {Project}'s user management.
+For example, a user belonging to the _internal_ location is unable to access a DNS service that is exclusively assigned to the _DMZ_ location.
```

Comment:
I think that a more robust set of use cases might really help hammer this home for users. 

@ezr-ondrej can you help with this as well? Could you provide perhaps 5 examples of where you've seen this to catch people?

Reply:
Well I guess main examples I have in mind are during provisioning and HG vs Host mismatches.

Hostgroup can have multiple orgs/locs so it can in theory have assigned subnet from one loc/org combination and Compute resource from a diferent combination and Host from this HostGroup is invalid by default.

If users on the other hand want to use organizations to separate all resources, there are some, that will stand in a way (e.g. Ansible roles, Operating system)

Reply:
I've reworked the introduction a bit and added your examples. Please re-review this.